website/data-storage/package.json

```diff
@@ -65,9 +69,9 @@
     "supertest": "6.2.3",
     "ts-jest": "27.1.5",
     "ts-loader": "8.4.0",
-    "ts-node": "9.1.1",
+    "ts-node": "^10.8.1",
```

Comment:
pls remove `^` because renovate-bot don't understand this

Reply:
Alright